tests/test_gamecontroller.py

```diff
@@ -25,41 +24,33 @@ def test_constructor_fail_too_many_players(self):
         assert str(excinfo.value) == 'There must be between 2 and 5 players to play Hanabi.'
 
     def test_deal_initial_hand_2p(self):
-        mock_deck = create_autospec(Deck)
-        mock_deck.draw_card.return_value = "card"
         mock_player = create_autospec(Player)
         gc = GameController([mock_player, mock_player])
-        gc.deck = mock_deck
+        gc.deck = util.create_mock_deck(r_draw_card="card")
```

Comment:
I added the 'r_' prefix to parameters that create a return value in the corresponding mock object. Not sure if that is useful. I figured that it's easy to remove.